llrt_modules/Cargo.toml

```diff
@@ -59,6 +61,11 @@ rquickjs = { version = "0.6", features = [
 ring = { version = "0.17", optional = true }
 tokio = { version = "1", features = ["rt", "io-util"] }
 tracing = "0.1"
+flate2 = { version = "1.0.30", features = [
```

Comment:
All those deps need to be optional and enabled by the feature

Reply:
Cargo.toml was modified.